14/alpine3.14/Dockerfile

```diff
@@ -0,0 +1,98 @@
+FROM alpine:3.14
+
+ENV NODE_VERSION 14.17.1
```

Comment:
Sorry, there was another release over the weekend if you can update this and the 16 version and CHECKSUM